src/cogs/fun.py

```diff
@@ -5,64 +5,60 @@
 from discord.ext import commands
 from datetime import datetime
 
+MINECRAFT_ENDPOINT = "https://api.mcsrvstat.us/2/{}"
 
 class Fun(commands.Cog):
-    """A Cog dedicated to various Fun Commands"""
+    """A cog dedicated to various fun commands such as 8 ball."""
     def __init__(self, bot):
         self.bot = bot
 
-    @commands.command(name="8ball",
-                      aliases=["eight_ball", "eightball", "8-ball"],
-                      )
-    async def eight_ball(self, ctx):
-        """Answers a yes/no question."""
-        possible_responses = [
-            "That is a resounding no",
-            "It is not looking likely",
-            "Too hard to tell",
-            "It is quite possible",
-            "Definitely",
-            "Why don\'t you figure it out yourself you lazy motherfucker",
-            "Yes, but do it drunk as fuck!",
-            "My sources say no, but they also said Hillary would win.",
-            "Do not swipe right, it\'s your cousin",
-            "Do swipe right, it\'s your cousin!",
-            "Do what Jesus would do, die at the age of 33",
-            "Trump uses me when deciding to go to war"
-
-        ]
-        em = discord.Embed(color=self.bot.config["colors"]["default"])
-        em.title = ":mage: The 8 ball wizard says..."
-        em.description = (random.choice(possible_responses) + ", " + ctx.author.mention)
+        with open("res/dadjokes.txt").readlines() as jokes:
+            self.jokes = jokes
+
+        with open("res/8ball.txt").readlines() as _8ball_responses:
+            self._8ball_responses = _8ball_responses
```

Comment:
I think this needs some error handling to cover the case of the files not being accessible.